installer/datafiles/Base_SCXCore.data

```diff
@@ -208,12 +208,13 @@ HandleConfigFiles() {
     rm -f /etc/opt/microsoft/scx/conf/cimserver_current.conf* /etc/opt/microsoft/scx/conf/cimserver_planned.conf* /etc/opt/microsoft/scx/conf/omiserver.conf*
 
     # File /etc/scxagent-enable-port opens port 1270 for usage with opsmgr
-    if [ -f /etc/scxagent-enable-port ]; then
+    if [ -f /etc/scxagent-enable-port ] || [ -f /etc/opt/microsoft/scx/conf/scxagent-enable-port ]; then
```

Comment:
This is in Base, not in a Linux-specific file, thereby involving UNIX. Is that the behavior you want? I don't necessarily think it's wrong, but we should think about this, as UNIX doesn't currently have this issue (Azure doesn't support it).